pkgs/tools/X11/kdocker/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, lib
+, qtbase
+, wrapQtAppsHook
+, fetchFromGitHub
+, qmake
+, qtx11extras
+, libX11
+, libXmu
+, libXpm
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "kdocker";
+  version = "5.4";
+
+  src = fetchFromGitHub {
+    owner = "user-none";
+    repo = "KDocker";
+    rev = "${finalAttrs.version}";
+    hash = "sha256-CTz2M9nv5Rf1amnSpLiIUZLH9Q3te6ZyFNUzSGHdYJc=";
+  };
+
+  nativeBuildInputs = [
+    qmake
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    qtbase
+    qtx11extras
+    libX11
+    libXmu
+    libXpm
+  ];
+
+  prePatch = ''
+    for h in Xatom Xlib Xmu; do
+      sed -i "s|#include <$h|#include <X11/$h|" src/xlibutil.h src/{kdocker,scanner,trayitem,trayitemmanager}.cpp
+    done
+    for t in target icons desktop appdata; do
+      sed -i "s|$t.path = /usr|$t.path = $out|" kdocker.pro
+    done
+    sed -i "s|/etc/bash_completion.d|$out/share/bash-completion/completions|" kdocker.pro
+  '';
+
+  meta = with lib; {
```

Comment:
Consider adding yourself as maintainer in here.

Reply:
But it already says `maintainers = with maintainers; [ hexclover ];` at line 50...?

Reply:
Oops my bad. Sorry !

Could you please sort alphabetically the meta attributes?

Reply:
Done!